run_kraken.py

```diff
@@ -49,50 +50,77 @@ def main(cfg):
         with open(cfg, "r") as f:
             config = yaml.full_load(f)
         global kubeconfig_path, wait_duration, kraken_config
-        distribution = config["kraken"].get("distribution", "openshift")
+        distribution = get_yaml_item_value(
+            config["kraken"], "distribution", "openshift"
+        )
         kubeconfig_path = os.path.expanduser(
-            config["kraken"].get("kubeconfig_path", "")
+            get_yaml_item_value(config["kraken"], "kubeconfig_path", "")
         )
         kraken_config = cfg
-        chaos_scenarios = config["kraken"].get("chaos_scenarios", [])
-        publish_running_status = config["kraken"].get("publish_kraken_status", False)
+        chaos_scenarios = get_yaml_item_value(
+            config["kraken"], "chaos_scenarios", []
+        )
+        publish_running_status = get_yaml_item_value(
+            config["kraken"], "publish_kraken_status", False
+        )
         port = config["kraken"].get("port")
```

Comment:
getting this error in the CI tests, lets go ahead and also set some sort of default value, maybe -1? @chaitanyaenr thoughts on default value? 

```
Traceback (most recent call last):
  File "/home/runner/work/krkn/krkn/run_kraken.py", line 517, in <module>
    main(options.cfg)
  File "/home/runner/work/krkn/krkn/run_kraken.py", line 165, in main
    if not 0 <= int(port) <= 65535:
TypeError: int() argument must be a string, a bytes-like object or a number, not 'NoneType'
```

Reply:
Yes, we can set it to 8081 -  default in the krkn config.